### PR TITLE
added gain to rms

### LINKED ISSUE #124: Add gain to RMS
Need to add a gain field to the RMS class, which will allow us to elegantly boost high frequencies and account for sensitivity of different microphones.

The class constructor and unit tests will need to be updated to achieve this.

### PR DESCRIPTION
**Pull request by Jambls**
Closes #124

### CODE REVIEW

File: include/rms.hh [hunk 1]
```diff
@@ -47,7 +50,8 @@ class RMS {
      *   @param output_pipe A Pipe<double> which will contain the root mean square of the signal
      *   @param timeout The number of milliseconds the thread should wait before exiting if no data is provided.
      */
-    RMS(int num_samples, Pipe<Audio> &input_pipe, Pipe<double> &output_pipe, std::chrono::milliseconds timeout);
+    RMS(int num_samples, double gain, Pipe<Audio> &input_pipe, Pipe<double> &output_pipe,
```
Comment (MHCooke) [inline include/rms.hh:53]: Remember to update the docs

File: unit_tests/rms_tests.cc [hunk 2]
```diff
@@ -88,12 +88,45 @@ TEST(RMSTests, multiple) {
...
+    ASSERT_EQ(rmsarr[0].calc(), 0);
+
+    audio[0] = 16;
+    for (int i = 0; i < 4; i++) {
+        rmsarr[i].insert(audio);
+    }
+    // buffer should now be 256, 0, 0, 0 with gain 0 giving rms of 0
+    ASSERT_EQ(rmsarr[0].calc(), 0);
+
+    // buffer should now be 256, 0, 0, 0 with gain 0.5 giving rms of 4/IM
```
[the remark this answers is not in the document]
Comment (Jambls) [reply unit_tests/rms_tests.cc:114]: It's concise. Is there something else you'd rather it was?

[the remark this answers is not in the document]
Comment (bear-in-the-air) [reply unit_tests/rms_tests.cc:114]: intmax or int16_max

**Review by MarkAHarley — APPROVED** [review]

**Review by MHCooke — APPROVED** [review]

**Review by bear-in-the-air — APPROVED** [review]

File: src/main.cc [hunk 3]
```diff
@@ -53,7 +53,7 @@ int main() {
         modulator_bank.at(i).run();
 
         // Power Meter
-        power_meter_bank.emplace_back(num_samples, modulator_out_pipes.at(i), power_out_pipes.at(i), timeout);
+        power_meter_bank.emplace_back(num_samples, 1, modulator_out_pipes.at(i), power_out_pipes.at(i), timeout);
```
Comment (MarkAHarley) [inline src/main.cc:56]: Make this 10 please or break it out to a variable at the top of main.

**Review by MarkAHarley — CHANGES_REQUESTED** [review]

File: src/main.cc (hunk 3 above)
Comment (Jambls) [reply src/main.cc]: Done

**Review by MarkAHarley — APPROVED** [review]

**Pull request merged**